django_projects/locallibrary/catalog/views.py

```diff
@@ -1,3 +1,44 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from django.views import generic
+from catalog.models import Book, Author, BookInstance, Genre
+from django.views.generic import DetailView
+def index(request):
+    """Hàm view cho trang chính của trang web."""
+    # Tạo số lượng của một số đối tượng chính
+    num_books = Book.objects.all().count()
```

Comment:
loại bỏ code dư thừa nhen
```suggestion
    num_books = Book.objects.count()
```